x/gov/migrations/v3/convert.go

```diff
@@ -54,9 +54,13 @@ func ConvertToLegacyProposal(proposal v1.Proposal) (v1beta1.Proposal, error) {
 				return v1beta1.Proposal{}, err
 			}
 			legacyProposal.Content = legacyMsg.Content
+			return legacyProposal, nil
 		}
 	}
-	return legacyProposal, nil
+	// hack to fill up the content with the first message
+	legacyProposal.Content, err = codectypes.NewAnyWithValue(msgs[0])
+
```

Comment:
The first idea I had, was to create a simple message:
```
message BasicContent {
  string title = 1;
  string description = 1 ;
}
```
and fill it using the `Content` interface. But, then we will remove additional attributes, which are in fact used by explorers to display the full content (I verified that with ping pub).

Reply:
I agree, the current way you have is best imo.